molecule/driver/openstackdriver.py

```diff
@@ -64,6 +84,13 @@ def platform(self):
     def platform(self, val):
         self._platform = val
 
+    def molecule_generated_keypair(self):
+        return 'keypair' not in self.molecule.config.config['openstack']
+
+    @property
+    def molecule_generated_ssh_key(self):
+        return self._molecule_generated_ssh_key
+
```

Comment:
I don't think a property is really needed.  We generally use properties in our drivers for abstract properties that are common across all drivers.